docs/OV_Runtime_UG/supported_plugins/GPU.md

```diff
@@ -36,122 +42,179 @@ Available devices:
 
-> **NOTE**: For operation descriptions, see the [IR Notation Reference](../../ops/opset.md).
+## Supported features
 
-### Fusing Convolution and Simple Layers
+### Multi-device execution
+If a machine has multiple GPUs (for example integrated GPU and discrete Intel GPU), then any supported model can be executed on all GPUs simultaneously.
+This can be achieved by specifying `"MULTI:GPU.1,GPU.0"` as a target device.
 
-Merge of a Convolution layer and any of the simple layers listed below:
-- Activation: ReLU, ELU, Sigmoid, Clamp, and others
-- Depthwise: ScaleShift, PReLU
-- FakeQuantize
+@snippet snippets/gpu/compile_model.cpp compile_model_multi
 
-> **NOTE**: You can have any number and order of simple layers.
+See [Multi-device execution page](../multi_device.md) for more details.
 
-A combination of a Convolution layer and simple layers results in a single fused layer called
-*Convolution*:
-![conv_simple_01]
+### Automatic batching
+GPU plugin is capable of reporting `ov::max_batch_size` and `ov::optimal_batch_size` metrics with respect to the current hardware platform and model,
+thus automatic batching can be applied in cases when `ov::hint::performance_mode(ov::hint::PerformanceMode::THROUGHPUT)` is set
+or device is specified as `"BATCH:GPU"`.
 
+@sphinxdirective
+
+.. tab:: Batching via BATCH plugin
+
+    .. doxygensnippet:: docs/snippets/gpu/compile_model.cpp
+        :language: cpp
+        :fragment: [compile_model_batch_plugin]
 
-### Fusing Pooling and FakeQuantize Layers
+.. tab:: Bacthing via throughput hint
 
-A combination of Pooling and FakeQuantize layers results in a single fused layer called *Pooling*:
-![pooling_fakequant_01]
+    .. doxygensnippet:: docs/snippets/gpu/compile_model.cpp
+        :language: cpp
+        :fragment: [compile_model_auto_batch]
 
-### Fusing Activation Layers
+@endsphinxdirective
+
+See [Automatic batching page](../automatic_batching.md) for more details.
 
-Given the linear pattern, an Activation layer can be fused into other layers:
+### Multi-stream execution
+If either `ov::num_streams(n_streams)` with `n_streams > 1` or `ov::hint::performance_mode(ov::hint::PerformanceMode::THROUGHPUT)` property is set for GPU plugin,
+then multiple streams are created for the model. In case of GPU plugin each stream has its own host thread and associated OpenCL queue
+which means that incoming infer requests can be processed simultaneously.
 
-![fullyconnected_activation_01]
+> **NOTE**: Simultaneous scheduling of kernels to different queues doesn't mean that the kernels are actually executed in parallel on GPU device. The actual behvaior depends on the hardware architecture, and in some cases the execution may be serialized inside the GPU driver.
 
+Implementation of streams in GPU plugin supports weights memory sharing across streams, thus memory consnumption may be less comparing to the case when the same model is loaded multiple times to the same device.
```

Comment:
What about describing in a more direct way? such as...
`It is recommended to use multi-stream feature when you need to run multiple inferences of same model in parallel because ...`

Reply:
Rephrased